### DISCUSSION TITLE
How to get pasted images work?

### DISCUSSION THREAD

**Discussion by wzsRQQjaYN** [maximevaillancourt/digital-garden-jekyll-template#168] [Q&A]
most of the time I need to clip-shot a thing and then paste it into an appropriate note in obsidian. When I do it, obsidian renders it well. but, after the git push, the site generated by Jekyll doesn't show the image, instead, it says it's an invalid link or something. 

I think it reads pasted images as "literal links", is this what's going on here? how to solve this?

#31 seems relevant but I can't figure out the actual solution, can anyone help, please?

**Comment by maximevaillancourt** [comment]
@MikeTannenbaum, you still use Obsidian with this garden template, right? (or at least, [you did at some point](https://refinedmind.co/obsidian-jekyll-workflow))

Do you know how to get images pasted in Obsidian to work with the Jekyll template by any chance? 🙏
